addons/im_livechat/static/src/js/ajax_external.js

```diff
@@ -0,0 +1,14 @@
+odoo.define('web.ajax_external', function (require) {
+"use strict";
+
+var ajax = require('web.ajax');
+
+/**
+  * This file should be used in the context of an external widget loading (e.g: live chat in a non-Odoo website)
+  * It overrides the 'loadJS' method that is supposed to load additional scripts, based on a relative URL (e.g: '/web/webclient/locale/en_US')
+  * As we're not in an Odoo website context, the calls will not work, and we avoid a 404 request.
+  */
+ajax.loadJS = function () {
```

Comment:
Looks like you forgot the `url` parameter.

Can you re-test it real quick to see if it logs correctly in the console and everything?

Reply:
@awa-odoo oups sorry :/ I wanted to go too fast